action-scripts/brownie/scripts/report_gauge_add.py

```diff
@@ -0,0 +1,165 @@
+import os
+from typing import Optional
+
+from bal_addresses import AddrBook
+from brownie import Contract
+from brownie import network
+from brownie.exceptions import VirtualMachineError
+from web3 import Web3
+
+from .script_utils import format_into_report
+from .script_utils import get_changed_files
+from .script_utils import get_pool_info
+
+ADDR_BOOK = AddrBook("mainnet")
+FLATBOOK = ADDR_BOOK.flatbook
+ROOT_DIR = os.path.dirname(
+    os.path.dirname(os.path.dirname(os.path.dirname(os.path.abspath(__file__))))
+)
+GAUGE_ADD_METHODS = ['gauge', 'rootGauge']
+
+STYLE_MAINNET = "Mainnet"
+STYLE_SINGLE_RECIPIENT = "Single Recipient"
+STYLE_CHILD_CHAIN_STREAMER = "ChildChainStreamer"
+
+SELECTOR_MAP = {
```

Comment:
We should add zkevm and avalanche too.  I think in the case of new gauge adds with the adder v4 you can alwasy figure out which gauge it is by looking at the name of the add function called.